src/config/notifier/index.ts

```diff
@@ -1,4 +1,6 @@
 export type SupportedEventChannel = | 'API'
+export type SupportedEvent = | 'SMARTCONTRACT' | 'NEWBLOCK'
```

Comment:
I think it would be neater to use const assertion here to mimic enum. That way you don't have to cast it upon usage (which remains same otherwise) and "intellisense" will give you options`.
```suggestion
export type SupportedEventType = keyof typeof SUPPORTED_EVENTS
```
where supported events is defined as:
```
export const SUPPORTED_EVENTS: { 
   SMARTCONTRACT: 'SMARTCONTRACT',
   NEWBLOCK: 'NEWBLOCK' 
} as const
```

Reply:
maybe do the same for SUPPORTED_EVENT_CHANNELS should you choose to go this route.

Reply:
done